dandi/tests/test_utils.py

```diff
@@ -218,7 +241,6 @@ def test_get_instance_cli_version_too_old():
             "services": {
                 "girder": {"url": "https://girder.dandi"},
                 "webui": {"url": "https://gui.dandi"},
-                "api": {"url": "https://publish.dandi/api"},
```

Comment:
note that redirector now will return `"api": None`

Reply:
Updated.